packages/reakit/src/Disclosure/DisclosureState.ts

```diff
@@ -98,7 +98,20 @@ export function useDisclosureState(
   React.useEffect(() => {
     if (typeof animated === "number" && animating) {
       setTimeout(() => setAnimating(false), animated);
+      return () => {};
     }
+    if (animated && animating && process.env.NODE_ENV === "development") {
+      const timeout = setInterval(() => {
+        console.warn(
```

Comment:
It should use `warning` from the `reakit-warning` package: https://github.com/reakit/reakit/tree/master/packages/reakit-warning